springboard_iogt/utils.py

```diff
@@ -100,6 +101,18 @@ def known(cls, indexes, localizer=None):
                 if cls.exists(section.get('name'), indexes)]
 
     @classmethod
+    def get_index(cls, name, indexes):
+        [index] = [index for index in indexes if name in index]
```

Comment:
I'm a bit confused. I would have expected the `if cls.exists(section.get('name'), indexes)` to cover cases where the index doesn't exist in `known_indexes()`. Was wondering more about cases where we have multiple indices that match `name in index`.

I don't think I have enough context to know whether the API should be returning an error for these cases, or whether it is not an error case if there are 0 or multiple indices matching a section name. I'm going to assume the latter for now, and make sure those cases are handled.